### DISCUSSION TITLE
Autocomplete renderInput prop

### DISCUSSION THREAD

**Discussion by jbournonville** [lookfirst/mui-rff#640] [Q&A]
Hello, i can't use the `renderInput` prop on Autocomplete component as in standard MUI-v5 field. I saw that you explicitly omit `renderInput` on `AutocompleteProps` declaration and I wonder why. 

Can you tell me the reason and an alternative to customize Autocomplete input?

BR

**Comment by lookfirst** [comment]
Thanks for the question. This could be an oversight as part of the v4->v5 migration, but I can't remember exactly why it was that way any more. =)

**Reply by jbournonville** [reply]
For mui v5, renderInput is required by default. (https://mui.com/api/autocomplete/)

I've made a quick fix with : 
```typescript
declare module "mui-rff" {
    interface AutocompleteProps<T, Multiple extends boolean | undefined, DisableClearable extends boolean | undefined, FreeSolo extends boolean | undefined> {
        renderInput: ( params: AutocompleteRenderInputParams ) => React.ReactNode
    }
}

```
With that Autocomplete work perfectly.

Can you consider removing `renderInput` omission ?

**Reply by svbrunov** [reply]
To whom it may concern,

I have just faced the same issue.

Using the following npm packages:

* `"mui-rff": "5.0.0"`.
* `"@mui/material": "5.4.3"`.

Best regards,
Sergey Vyacheslavovich Brunov.

**Reply by lookfirst** [reply]
@svbrunov Sorry to hear that you're facing the same issue. Would you like to try to work on a PR to fix it?

**Reply by svbrunov** [reply]
Dear Public Profile (@lookfirst),

Sorry for bothering you!

I have found [the properties](https://github.com/lookfirst/mui-rff/blob/v5.0.0/src/Autocomplete.tsx#L158-L159) I was looking for.
Therefore, I don't have the issue.

Best regards,
Sergey Vyacheslavovich Brunov.
